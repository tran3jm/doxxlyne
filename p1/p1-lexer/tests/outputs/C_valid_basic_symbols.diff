--- outputs/C_valid_basic_symbols.txt	2022-09-11 14:08:08.976805361 -0400
+++ expected/C_valid_basic_symbols.txt	2022-09-11 13:08:06.057822040 -0400
@@ -0,0 +1,7 @@
+DECLIT   [line 001]  39
+ID       [line 001]  skdnbsljfd
+ID       [line 002]  asd
+ID       [line 002]  slj
+DECLIT   [line 002]  203
+ID       [line 002]  c
+HEXLIT   [line 003]  0x348
\ No newline at end of file
