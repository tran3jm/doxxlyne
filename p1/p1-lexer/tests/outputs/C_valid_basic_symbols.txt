SYMBOL   [line 001]  (
SYMBOL   [line 002]  )
SYMBOL   [line 003]  +
SYMBOL   [line 004]  !
SYMBOL   [line 005]  ?
SYMBOL   [line 006]  *
SYMBOL   [line 007]  :
SYMBOL   [line 008]  ;
