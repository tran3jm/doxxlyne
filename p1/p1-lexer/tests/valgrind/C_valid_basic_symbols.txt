==3456343== Memcheck, a memory error detector
==3456343== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3456343== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3456343== Command: ../decaf inputs/valid_symbols_c.decaf
==3456343== 
Invalid token!
==3456343== 
==3456343== HEAP SUMMARY:
==3456343==     in use at exit: 24,560 bytes in 285 blocks
==3456343==   total heap usage: 338 allocs, 53 frees, 41,108 bytes allocated
==3456343== 
==3456343== LEAK SUMMARY:
==3456343==    definitely lost: 400 bytes in 7 blocks
==3456343==    indirectly lost: 24,160 bytes in 278 blocks
==3456343==      possibly lost: 0 bytes in 0 blocks
==3456343==    still reachable: 0 bytes in 0 blocks
==3456343==         suppressed: 0 bytes in 0 blocks
==3456343== Rerun with --leak-check=full to see details of leaked memory
==3456343== 
==3456343== For lists of detected and suppressed errors, rerun with: -s
==3456343== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
