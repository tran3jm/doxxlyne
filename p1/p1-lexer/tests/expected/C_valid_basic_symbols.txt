DECLIT   [line 001]  39
ID       [line 001]  skdnbsljfd
ID       [line 002]  asd
ID       [line 002]  slj
DECLIT   [line 002]  203
ID       [line 002]  c
HEXLIT   [line 003]  0x348